lib/http/chainable.rb

```diff
@@ -6,42 +6,48 @@
 
 module HTTP
   module Chainable
-    # Request a get sans response body
+    # Request a get sans response body.
+    # See Client#request.
```

Comment:
Please use `@see` tag

```suggestion
    # @see Client#request
```